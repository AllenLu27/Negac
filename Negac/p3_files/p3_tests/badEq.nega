void main(){
	int a;
	int b;
	if (a != false){
	}
}
