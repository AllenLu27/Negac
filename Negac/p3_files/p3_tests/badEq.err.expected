FATAL [4,8]: Invalid equality operation
Semantic analysis failed
